File: 2° semestre/Técnicas de programação/aula-24-04-12/atividade5/src/Funcionario.java
public class Funcionario {
    String nome;
    double salarioBruto;
    double desconto;
    double salarioAjustado;

    public double salarioLiquido() {
        double sl;
        sl = salarioBruto - desconto;
        return sl;
    }

    public void aumentarSalario(double porcent) {
        double sl = salarioLiquido();
        salarioAjustado = sl + (sl / porcent);
    }
}

File: 3° semestre/Programação-dispositivos-moveis/aula-24-09-06/src/Funcionario.kt
class Funcionario(){
    var nome: String = ""
    var salarioBruto: Double = 0.0
    var desconto: Double = 0.0

    fun salarioLiquido(): Double{
        return (salarioBruto - desconto)
    }

    fun aumentarSalario(porcent:Double): Double{
        return salarioLiquido() + (salarioBruto * porcent / 100)
    }
}

